src/components/CustomIcon.tsx

```diff
@@ -628,8 +628,8 @@ const CustomIconMap = {
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
-        fill-rule="evenodd"
-        clip-rule="evenodd"
```

Comment:
Oops that was me

Reply:
Hah all good! @lee-at-zoo-corp pointed it out to me